packages/esm-service-queues-app/src/hooks/useQueueEntries.ts

```diff
@@ -81,11 +82,39 @@ export function useQueueEntries(searchCriteria?: QueueEntrySearchCriteria, rep:
   const [data, setData] = useState<Array<Array<QueueEntry>>>([]);
   const [totalCount, setTotalCount] = useState<number>();
   const [currentPage, setCurrentPage] = useState<number>(0);
+  const [searchCriteria, setSearchCriteria] = useState(searchCriteriaVal);
```

Comment:
I would rather have it

```suggestion
  const [currentSearchCriteria, setCurrentSearchCriteria] = useState(searchCriteria);
```

Since the suffix "val" is meaningless

Reply:
(ditto for `repVal`)

Reply:
Also the names of the arguments constitute the API of the function, which means they are more important names than the ones in the implementation details

Reply:
Duly noted, thanks!